modules/gitops-release-notes-1-1.adoc

```diff
@@ -0,0 +1,129 @@
+
+- *GA*: _General Availability_
+
+.GitOps Technology Preview features
+[cols="1,1",options="header"]
+|===
+| {gitops-title} Product | Release
+| Argo CD
+| GA
+| Argo CD ApplicationSet
+| TP
+| {gitops-title} Application Manager (kam)
+| TP
+| {gitops-title} Service
+| TP
+|===
+
+
+[id="new-features-1-1_{context}"]
+== New features
+{gitops-title} 1.1 is now available on {product-title} 4.7. These are the new features in {gitops-title} 1.1:
+
+* The `ApplicationSet` feature is now added (Technology Preview). The `ApplicationSet` feature enables both automation and greater flexibility when managing Argo CD applications across a large number of clusters and within monorepos. It also makes self-service usage possible on multitenant Kubernetes clusters.
+* Argo CD is now integrated with cluster logging stack and with the {product-title} Monitoring and Alerting features.
+* Argo CD auth is now integrated with {product-title}.
+* Argo CD applications controller now supports horizontal scaling.
+* Argo CD Redis servers now support high availability (HA).
+
+In addition to the new features, the following sections highlight the fixes and stability improvements in {gitops-title} 1.1.:
+
+[id="fixed-issues-1-1_{context}"]
+== Fixed issues
+The following issues been resolved in the current release:
+
+* Previously, {gitops-title} did not work as expected in a proxy server setup with active global proxy settings. This issue is fixed and now Argo CD is configured by the {gitops-title} Operator using fully qualified domain names (FQDN) for the pods to enable communication between components. link:https://issues.redhat.com/browse/GITOPS-703[GITOPS-703]
+* The {gitops-title} backend relies on the `?ref=` query parameter in the {gitops-title} URL to make API calls. Previously, this parameter was not read from the URL, causing the backend to always consider the default reference. This issue is fixed and the {gitops-title} backend now extracts the reference query parameter from the {gitops-title} URL and only uses the default reference when there is no input reference provided. link:https://issues.redhat.com/browse/GITOPS-817[GITOPS-817]
+* Previously, the {gitops-title} backend failed to find the valid GitLab repository. This was because the {gitops-title} backend checked for `main` as the branch reference, instead of `master` in the GitLab repository. This issue is fixed now. link:https://issues.redhat.com/browse/GITOPS-768[GITOPS-768]
+* The *Environments* page in the {product-title} web console from the *Developer* perspective now shows the list of applications and the number of environments along with an Argo CD link that directs you to the Argo CD *Applications* page that lists all the applications. The Argo CD *Applications* page has *LABELS* (for example, `app.kubernetes.io/name=appName`) that helps you filter only the applications of your choice to be shown in that view. link:https://issues.redhat.com/browse/GITOPS-544[GITOPS-544]
```

Comment:
Split the first sentence it is too huge: The *Environments* page in the {product-title} web console from the *Developer* perspective now shows a list of applications and the number of environments. It also displays an Argo CD link that directs you to the Argo CD *Applications* page that lists all the applications.
helps/help
remove `to be shown in that view`

Reply:
Done